.github/workflows/flutter_package.yml

```
@@ -80,6 +83,14 @@ jobs:
           cache: true
           cache-key: flutter-:os:-:channel:-:version:-:arch:-:hash:-${{ hashFiles('**/pubspec.lock') }}
 
+      - name: 🤫 Set SSH Key
+        env:
+          ssh_key: ${{secrets.ssh_key}}
+        if: env.ssh_key != null
```

Can't we just do this:
```suggestion
	if: "${{secrets.ssh_key != ''}}"
```

Instead of storing it in ENV?

---

@wolfenrain  That was my first approach to. But you cannot access the secrets inside the `if` condition. [Here's](https://github.com/orgs/community/discussions/26726) a discussion around that topic.